src/io/result_io.jl

```diff
@@ -68,17 +68,20 @@ N is the number of dhw/wave scenarios.
 function store_env_summary(data_cube::NamedDimsArray, type::String, file_loc::String, rcp::String, compressor::Zarr.Compressor)::ZArray
     stats = summarize_env_data(data_cube)
 
-    stats_store = zcreate(Float32, (2, size(stats, 2))...;
+    stats_store = zcreate(
+        Float32,
```

Comment:
`summarize_env_data` creates an `Array{Float64}` and here you are creating a `ZArr{Float32}`. Just want to doublecheck check if maybe you want to change one of them..?  (I know there may be reasons for doing this so, if this is right, feel free to ignore this comment).

Reply:
Not sure, this was in Takuya's original function- @ConnectedSystems is there a reason for one being Float32 and one Float64?

Reply:
Data is stored as Float32 to save on space. This was an early decision that I ran by the team 2+ years ago. We lose precision by doing this of course, but not sure if it matters.

FYI - The RM team also store all their results as single precision values.

Reply:
Thanks Takuya